packages/lesswrong/components/languageModels/LlmConversationsViewingPage.tsx

```diff
@@ -100,22 +101,25 @@ const LlmConversationRow = ({conversation, currentConversationId, setCurrentConv
   setCurrentConversationId: (conversationId: string) => void,
   classes: ClassesType<typeof styles>,
 }) => {
+  const { LWTooltip, FormatDate, UsersNameDisplay } = Components;
   const isCurrentlySelected = currentConversationId === conversation._id;
   const { title, user, lastUpdatedAt, createdAt } = conversation;
 
-  return <div 
-    className={classNames(classes.conversationRow, {[classes.conversationRowSelected]: isCurrentlySelected})}
-    onClick={()=> setCurrentConversationId(conversation._id)}
-    >
+  const conversationCharacterCount = conversation.totalCharacterCount ?? 0;
+  const estimatedTokenCount = Math.round(conversationCharacterCount / 4.4);
+
+  return <div className={classNames(classes.conversationRow, {[classes.conversationRowSelected]: isCurrentlySelected})}>
     <span className={classes.conversationRowGroup}>
-      <span className={classes.conversationRowUsername}>{userGetDisplayName(user)}</span>
-      <span className={classes.conversationRowTitle}>{title}</span>
+      <UsersNameDisplay user={user} className={classes.conversationRowUsername} hideFollowButton />
+      <span className={classes.conversationRowTitle} onClick={()=>setCurrentConversationId(conversation._id)}>
+        <Link to={`/admin/llmConversations?conversationId=${conversation._id}`}>{title}</Link>
```

Comment:
I think you probably don't need both the `setCurrentConversationId` and the `Link` to get the effect you want, but can come back to it

Reply:
I tried just having the Link and stuff broke when modifying useEffect, and I was "figure out another time"